connect/runtime/src/test/java/org/apache/kafka/connect/util/clusters/EmbeddedKafkaCluster.java

```diff
@@ -254,7 +254,7 @@ public void produce(String topic, Integer partition, String key, String value) {
         try {
             producer.send(msg).get(DEFAULT_PRODUCE_SEND_DURATION_MS, TimeUnit.MILLISECONDS);
         } catch (Exception e) {
-            throw new KafkaException("Could not produce message to topic=" + topic, e);
+            throw new KafkaException("Could not produce message: " + msg, e);
```

Comment:
Can we still include the topic name in the exception?

Reply:
the topic name is included in the msg (that's why I removed it here).

Reply:
Ah, nevermind, the topic is in the message's `toString()`.